src/services/authService.js

```diff
@@ -29,6 +29,9 @@ export const  CreateReport = (data) => {
 export const  GetReportByID = (ReportID) => {
 	return http.get(apiReportUrl + ReportID);
 }
+export const  GetReportComments = (ReportID) => {
```

Comment:
can be simplified to 

```js
export const  GetReportComments = (ReportID) => http.get(apiReportUrl + ReportID + '/comments');
```

Reply:
This I'm not sure what the difference is. Did you remove the return and move it all to one line? Don't you need the return to populate the response from the call? And if the return was removed by accident do you just want this on one line?

Reply:
```js
export const  GetReportComments = (ReportID) => {
 return ...
}
```

is the same as

```js
export const  GetReportComments = (ReportID) => http.get(apiReportUrl + ReportID + '/comments');
```
in this case each will return the Promise returned by axios

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Basic_syntax

it's not anything that well die if we don't do, but aligns with the styleguide we have been using.

https://github.com/airbnb/javascript#arrow-functions

Reply:
Okay no worries. I wasn't sure exactly what you meant. I'll update.